### DISCUSSION TITLE
Should religion be subclass of ideology?

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#403] [Announcements]
Ran across this while showing CCO to colleagues.

**Comment by swartik** [comment]
For what it's worth, I would say no according to two dictionary definitions.

- The OED's definition says that ideology usually relates "to politics, economics, or society and forming the basis of action or policy". 
- Merriam Webster gives 3 definitions. One is plausibly a parent of religion. Another says ideology relates to sociopolitical programs. The third says ideology is a body of concepts about _human_ life. No mention of the supernatural.

"Religion" has a source, but "Ideology" doesn't. CCO authors, do you recall where Ideology's definition came from?

**Comment by alanruttenberg** [comment]
If that's the case then I think the definition could be adjusted or a scope note added on each to clarify. The first reaction of the colleague I was talking with was that, based on the current definitions, it looks like it should be subclass, and I concurred.

**Comment by bdonohue29** [comment]
In ordinary language, I would not say "a religion is an ideology," because of the reasons offered above by swartik (connection to sociopolitical concerns, activism, etc).

So I guess this hinges on: What's the intent behind the CCO_0005041 class (labeled "Ideology")?
- If the intent is to capture the notion of ideology in the narrower sense (pertaining to politics, economics, society, etc), then I'd recommend an update to the textual definition to reflect this.
- If the intent is just to capture the notion of ideology in a much broader sense (roughly "any worldview"), then I'd suggest an update to the main label.
